pkg/virtctl/create/clone/clone.go

```diff
@@ -23,14 +23,15 @@ import (
 	"fmt"
 	"strings"
 
-	"kubevirt.io/kubevirt/pkg/pointer"
-
 	"github.com/spf13/cobra"
```

Comment:
it's not related with this PR but, newline?

Reply:
I moved this local import to the bottom of the imports on purpose.

Reply:
Sorry, I didn't explain myself properly. I meant between:

Reply:
Those are all remote imports, so AFAIK we don't separate them further.